uaclient/security_status.py

```diff
@@ -26,20 +30,23 @@ class UpdateStatus(Enum):
     UNAVAILABLE = "upgrade_unavailable"
 
 
-def get_service_name(origins: List[apt_package.Origin]) -> str:
+def is_esm_package(package: apt_package.Package, service_name: str):
+    return any(
```

Comment:
Instead of using `any` that will evaluate all the origins, we can have a standard loop that will break if the condition we want is met. In that way, we may not need to go through all the origins

Reply:
Most packages will have 2-3 origins so it is not a big deal to make the list; and `any` short circuits anyway.
Not a strong opinion on this though, can change np

Reply:
Oh okay, TIL that it does short circuit, so no problem at all

Reply:
Ended up that I rewrote it to only go through origins once, as it costs more than checking the strings in the caller - small improvement (: